.github/workflows/testing.yml

```diff
@@ -43,35 +43,35 @@ jobs:
       matrix:
         include:
           - type: vet
-            goversion: '1.22'
+            goversion: '1.23'
```

Comment:
I thought we agreed to keep vet running at 1.22?

Reply:
You're right, reverted to 1.22 for vet. There is a problem in vet.sh where it sets `PATH` using `GOROOT` even when `GOROOT` is unset. Because of that, vet was using go1.23. I've made the fix in this PR.